src/util/embedControl.js

```diff
@@ -0,0 +1,50 @@
+L.Control.Embed = L.Control.extend({
+
+    options: {
+        position: 'bottomleft',
+    },
+
+    initialize: function(options) {
+        L.Util.setOptions(this, options);
+        this._embedElement = L.DomUtil.create('button');
+        this._embedIconElement = L.DomUtil.create('i', 'fa fa-code')
+        this._embedElement.appendChild(this._embedIconElement);
+        this.onClick();
+    },
+
+    onAdd: function(map) {
+        return this._embedElement;
+    },
+
+    onClick: function() {
+        var self = this;
+        L.DomEvent.on(this._embedElement, 'click', function(ev) {
+            prompt('Use this HTML code to embed this map on another site.', self.generateCode())
+        })
+    },
+
+    getUrlHashParameters: function() {
+        var hash = window.location.hash;
+        var getValues = hash.substr(1).split('/');
+        var params = {
+            lat: getValues[1],
+            lng: getValues[2],
+            zoom: getValues[0],
+            layers: getValues[3]
+        }
+        return params;
+    },
+
+    generateCode: function() {
+        var params = this.getUrlHashParameters();
+        var code = '<iframe style="border:none;" width="100%" height="900px" src="//publiclab.github.io/leaflet-environmental-layers/example/#' + params.zoom + '/' + params.lat + '/' + params.lng + '/' + params.layers +'"></iframe>';
```

Comment:
Hey @crisner ,
Instead of this URL, let's use the one i created in PL:
https://publiclab.org/map/

Reply:
I tried loading the mentioned URL but I am getting a 404 page.